### ISSUE TITLE
Acronyms and abbreviations

### ISSUE THREAD

**Issue by jfhovinne** [wymeditor/wymeditor#22]
I think support for ''<abbr>'' and ''<acronym>'' with at least the ''title'' attribute should be implemented (editable) in WYMeditor. This would improve the semantics and accessibility of what the editor will output.
